src/fuzz_test.cpp

```diff
@@ -0,0 +1,26 @@
+#include <unistd.h>
+
+#include "main.hpp"
```

Comment:
You can `#include "main.cpp"` here.  That avoids a lot of churn.

Reply:
Not a bad idea but I'd have to conditionally rename main when fuzz test is selected.

Reply:
You mean this?
```c
#define main not_real_main
#include "main.cpp"
#undef main
```
Seems acceptable to me.